src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -125,4 +125,8 @@ public void linkOnCommentsLink() {
         AppPage.getNewPageInstance(browser, StudentCommentsPage.class);
         
     }
+    
+    public void waitForCellHoverToDisappear() {
+        waitForElementToDisappear(By.className("cell-hover"));
```

Comment:
Hmm, I was assuming it works by polling to see if the element with the class `cell-hover` is visible/present on the page. If the `cell-hover` class is removed from the element, then there no longer exists an element on the page with a `cell-hover` class, so essentially it "disappears".